pyrtls.pyi

```diff
@@ -0,0 +1,407 @@
+"""
+# pyrtls: rustls-based modern TLS for Python
+
+[Latest version](https://pypi.org/project/pyrtls) |
+[Documentation](https://pyrtls.readthedocs.io) |
+[CI](https://github.com/djc/pyrtls/actions?query=workflow%3ACI+branch%3Amain)
+
+pyrtls provides bindings to [rustls](https://github.com/rustls/rustls), a modern Rust-based TLS implementation with an API that is
+intended to be easy to use to replace the `ssl` module (but not entirely compatible with it).
+
+In addition to being memory-safe, the library is designed to be more secure by default. As such,
+it does not implement older protocol versions, cipher suites with known security problems, and
+some problematic features of the TLS protocol. For more details, review the [manual](https://docs.rs/rustls/latest/rustls/manual/index.html).
+
+## Warning!
+
+This project is just getting started. While rustls is mature, the Python bindings
+are pretty new and not yet feature-complete.
+"""
+
+from collections.abc import Iterable
+from socket import socket
+
+# lib.rs
+
+class TrustAnchor:
```

Comment:
Should TrustAnchor be included? It is annotated as #[pyclass], but it isn't included in the #[pymodule] function. I'm not that familiar with PYO3 so I'll leave that up to you.

Reply:
Yup, this was an oversight -- I added it in #52.

Reply:
Synced 👍